addon/components/editor.ts

```diff
@@ -111,6 +116,9 @@ export default class RdfaEditor extends Component<RdfaEditorArgs> {
       nodeViews: this.args.nodeViews,
       defaultAttrGenerators: this.args.defaultAttrGenerators,
       keyMapOptions: this.args.keyMapOptions,
+      editable: () => {
+        return !(this.editable === false);
```

Comment:
you can do !!this.editable I think

Reply:
Not in this case I think, as we do not want to trigger it when `editable` is `undefined` or `null`, this is why we strictly compare it to `false`.